src/pages/Possibility/KivaContentBlock.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+	<kv-carousel @change="slideChange">
+		<div class="slide-placeholder"></div>
+		<kv-carousel-slide :key="headline" v-for="{ images, url, headline, subheadline } in kivaContentBlockData">
```

Comment:
This will result in duplicate :keys. Could pass an index into the v-for and do something like :key="`headline-${index}`"

Reply:
Each of the headlines is unique, so there won't be a problem with duplicate keys for the current set of slides, but adding the index would be a way to future-proof this component in case slides are added that have the same headline.

Reply:
@ryan-ludwig Each of the headlines are unique. Just trying to understand why this will result in duplicate keys? Can you expand on this a little bit for me?

Reply:
Ah you're right, I was reading it as a string instead of a binding.

Reply:
Made these changes, thanks for the discussion guys.